omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/search/openmrs1_8/ConceptSearchHandler1_8.java

```diff
@@ -50,10 +50,14 @@ public class ConceptSearchHandler1_8 implements SearchHandler {
 	private final SearchConfig searchConfig = new SearchConfig("default", RestConstants.VERSION_1 + "/concept",
 	        Arrays.asList("1.8.* - 9.*"),
 	        Arrays.asList(
-	            new SearchQuery.Builder("Allows you to find concepts by source and code").withRequiredParameters("source")
-	                    .withOptionalParameters("code").build(), new SearchQuery.Builder(
-	                    "Allows you to find concepts by name and class").withRequiredParameters("name")
-	                    .withOptionalParameters("class", "searchType").build()));
+	            new SearchQuery.Builder("Allows you to find concepts by source and code")
+					.withRequiredParameters("source")
+					.withOptionalParameters("code")
+					.build(),
+				new SearchQuery.Builder("Allows you to find concepts by name and class")
+					.withOptionalParameters("class", "name", "searchType")
+					.build()
+			));
```

Comment:
Is this change part of the ticket requirement?

Reply:
Yes, I believe so. The ticket is to allow name to be optional instead of required. This patch moves the name parameter from required to optional along with the class and search type parameter ( with additional formatting added ).